modules/learn/pages/buckets-memory-and-storage/memory.adoc

```diff
@@ -57,13 +57,27 @@ Minimum-required memory-quotas (which are the quota-defaults initially provided
 == Initialization and Warmup
 
 When Couchbase Server is restarted on a node, the node goes through a _warmup process_ before it restarts the handling of data requests.
-During this warmup process, data on disk is sequentially reloaded into memory.
+During this warmup process, data on disk is sequentially reloaded into memory for each bucket.
 The time required for the reload depends on the size and configuration of the system, the amount of data persisted on the node, and the ejection policy configured for the buckets.
 
-Frequently used items are identified via a scanner process, which examines an _access log_, and obtains the appropriate keys.
-Corresponding items are then loaded with the highest priority.
-The scanner process is configurable, via the CLI utility `cbepctl`, with the `flush_param` parameter.
+On a given Data Service node, a bucket's data is loaded in accordance with determinations made by the _access scanner_.
+This program runs by default once every day.
+It checks the _resident ratio_ &#8212; which is the percentage, on the current node, of bucket-items currently in memory:
```

Comment:
Think it needs to be clear that this is the combined Resident Ratio across both Active and Replica vBuckets.

Reply:
Maybe rephrase to .... It checks the _resident ratio_ &#8212; which is the percentage of items in active and replica vBuckets that are currently in memory on that node.